preprocessing/training/03_convert_to_jsonl/get_dataset_name.py

```diff
@@ -1,8 +1,12 @@
 import argparse
+import re
 from pathlib import Path
+
 import pandas as pd
 
-from clean_helpers.utils import normalise_dataset_name_regex
+normalise_dataset_name_regex = re.compile(
+    r"^(?:/gpfswork/rech/six/uty16tp/dataset/tokenization/)?(bigscience-catalogue-lm-data/[^/]+)(?:/data)?$"
+)
```

Comment:
duplicated with somewhere else?

Reply:
Indeed, before it was imported from but I could not solve the import path with the reorganization, so I thought I could just copy and paste it